app/models/post.rb

```diff
@@ -0,0 +1,16 @@
+class Post < ApplicationRecord
+  belongs_to :author, class_name: 'User', foreign_key: 'author_id'
+  has_many :comments
+  has_many :likes
+  after_save :update_post_counter
+
+  def five_recent_comments
+    Comment.where(post_id: id).order(created_at: :desc).limit(5)
+  end
```

Comment:
[OPTIONAL]
- [ ] It's great that you're working with the data effectively! :clap: I'd like to suggest a more efficient approach for retrieving the last five comments associated with a post.

  The code `Comment.where(post_id: id).order(created_at: :desc).limit(5)` certainly works, but it can be further simplified. By using `comments.order(created_at: :desc).limit(5)`, you take advantage of the association between the post and its comments, which is generally more performant.